x-pack/plugins/lens/public/editor_frame_service/editor_frame/editor_frame.test.tsx

```diff
@@ -130,88 +192,95 @@ describe('editor_frame', () => {
     expressionRendererMock = createExpressionRendererMock();
   });
 
+  const mountWithProvider = async (
```

Comment:
I think we do the very same thing for the workspace panel (and we will probably need it in a lot of other places as well). Would it make sense to pull that one out?